clients/cli/src/clap_app.rs

```diff
@@ -2688,4 +2703,46 @@ pub fn app<'a>(
                 .arg(multisig_signer_arg())
                 .nonce_args(true)
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateUiMultiplier.into())
+                .about("Update UI multiplier")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(|s| is_valid_pubkey(s))
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The token address with scaled UI amount multiplier"),
+                )
+                .arg(
+                    Arg::with_name("ui-multiplier")
+                        .value_name("MULTIPLIER")
+                        .takes_value(true)
+                        .index(2)
+                        .required(true)
+                        .help("The new multiplier"),
+                )
+                .arg(
+                    Arg::with_name("timestamp")
+                        .value_name("TIMESTAMP")
+                        .takes_value(true)
+                        .index(3)
+                        .required(true)
+                        .help("The effective time for the new multiplier",)
```

Comment:
nit to be totally clear
```suggestion
                        .help("The effective time for the new multiplier, given as a UNIX timestamp",)
```